modules/luci-base/po/de/base.po

```diff
@@ -7538,7 +7538,7 @@ msgstr "schwerwiegende Fehlersekunden (SES)"
 #: modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js:210
 #: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/60_wifi.js:39
 msgid "Short GI"
-msgstr "kurzes Guardintervall"
+msgstr "GI curto"
```

Comment:
@SantosSi - seems a portuguese translation incorrectly slipped in here, could you quikcly fix it in weblate?